docs/UserGuide.md

```diff
@@ -196,8 +202,8 @@ _Details coming soon ..._
 | **delete**       | `delete MODULE_CODE` <br> e.g., `delete CS2040S`                                    |
 | **edit**         | `edit MODULE_CODE [y/YEAR] [s/SEMESTER] [g/GRADE]` <br> e.g., `edit MA2001 y/1 s/2` |
 | **info**         | `info MODULE_CODE`<br> e.g., `info CS3230`                                          |
-| **CalculateCap** | `CalculateCap`                                                                      | 
-| **CalculateMC**  | `CalculateMC`                                                                       |
+| **calculateCAP** | `calculateCap`                                                                      | 
```

Comment:
"calculateCap" results in "Unknown command" being displayed to the user. The correct format is "calculateCAP".

Reply:
Fixed